assets/js/script.js

```diff
@@ -33,11 +85,62 @@ function getForecast(city) {
         })
 };
 
+//Uses coordiates to display google map with traffic overlay
 function initMap(city) {
     const map = new google.maps.Map(document.getElementById("map"), {
       zoom: 13,
       center: { lat: city.coord.lat, lng: city.coord.lon },
     });
     const trafficLayer = new google.maps.TrafficLayer();
     trafficLayer.setMap(map);
-  }
+  };
+
+//startup process basied on parameters
+if (savedCity && userName) {
+    displayInfo(savedCity, userName);
+    getCityData(savedCity, "startup");
+} else if (!savedCity && !userName) {
+    savedCity = "";
+    userName = "";
+    modalMaker("both");
+} else if (!savedCity && userName) {
+    savedCity = "";
+    modalMaker("city");
+} else if (savedCity && !userName) {
+    userName = "";
+    modalMaker("name");
+    getCityData(savedCity, "startup");
+};
+
+$("form").on("submit", function (event) {
+    event.preventDefault();
+    var cityName = $("#city").val();
+    var name = $("#name").val();
+    localStorage.setItem('name', (name));
+    localStorage.setItem('city', cityName);
+    if (cityName) {
+        getCityData(cityName, "startup");
+    }
+    else {
+        var entryInvalidParagraph = document.querySelector('#invalid-entry-p');
+        if (!entryInvalidParagraph) {
+            var entryInvalidParagraph = document.createElement('p');
+            var formDiv = document.querySelector('#map');
+            entryInvalidParagraph.id = 'invalid-entry-p';
+            entryInvalidParagraph.textContent = 'Must fill out both forms';
+            entryInvalidParagraph.className = 'non-fading';
+            entryInvalidParagraph.className = 'fading';
+            formDiv.appendChild(entryInvalidParagraph);
+        }
+        else {
+            entryInvalidParagraph.innerHTML = '';
+            var formDiv = document.querySelector('#map');
+            entryInvalidParagraph.id = 'invalid-entry-p';
+            entryInvalidParagraph.textContent = 'Must fill out both forms';
+            entryInvalidParagraph.className = 'non-fading';
+            entryInvalidParagraph.className = 'fading';
```

Comment:
I believe this can be acieved with one line (entryInvalidParagraph.classname = "non-fading fading")